internal/cli/cmd/cluster/create.go

```diff
@@ -565,110 +556,6 @@ func (o *CreateOptions) buildComponents(clusterCompSpecs []appsv1alpha1.ClusterC
 	return comps, nil
 }
 
-const (
-	saNamePrefix          = "kb-sa-"
-	roleNamePrefix        = "kb-role-"
-	roleBindingNamePrefix = "kb-rolebinding-"
-)
-
-// buildDependenciesFn creates dependencies function for components, e.g. postgresql depends on
-// a service account, a role and a rolebinding
-func (o *CreateOptions) buildDependenciesFn(cd *appsv1alpha1.ClusterDefinition,
```

Comment:
This PR remove the creation and deletion of sa,rolebinding and role in CLI. 

Now the CLI has no way of knowing whether the controller has rbac permission to create rolebing and serviceaccount. If there is no permission, the CLI will not create rolebinding and service, which will trigger the failure of cluster creation.

For cluster helm chart, I will add a `serviceAccount.Enabled` field, and default to `false`, ignore it to create sa and rolebinding.

I suggest adding a permission check to the webhook. If there is no relevant permission, report an error as soon as possible and do not enter reconcile.

Reply:
In one sentence, the CLI and helm chart will not create rolebinding and sa by default, and the cluster creation may fail due to insufficient permissions. In the future, the CLI needs to provide a method for users to specify an existing sa or automatically create sa and rolebing when controller has insufficient permissions.

Reply:
👌🏻, move it back